### PR TITLE
feat(cli): add 'clear' command to clear the terminal and stay in loop

### LINKED ISSUE #139: clear cmd
it would be great if we have clear cmd in terminal
The clear command is a powerful addition to our CLI tool, significantly boosting productivity and improving the user experience.

### PR DESCRIPTION
**Pull request by mdex-geek**
fixes #139 
- Implemented a 'clear' command in the CLI to clear the terminal screen.
- The clear screen functionality is supported for both Windows ('cls') and Unix-based systems ('clear').
- The program remains in the command loop after clearing the screen, allowing continuous use until 'exit' is entered.

https://github.com/pieces-app/cli-agent/assets/107706485/55258614-c386-4152-bd00-9bd8dd85d033

### CODE REVIEW

**Comment by Bishoy-at-pieces** [comment]
@mdex-geek great job looks good but can you add this to the CHANGELOG.md? And the README.md too

**Comment by mdex-geek** [comment]
@Bishoy-at-pieces i add this in changelog.md and readme.md

**Comment by Bishoy-at-pieces** [comment]
Thank you!

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,3 +15,7 @@
 - Fix some bugs
 - Add edit --classifcation  to reclassify an asset
 - Add the save asset command
+
+## March 25th,2024
+
+- add new cmd for  CLI
```
Comment (Bishoy-at-pieces) [inline CHANGELOG.md:21]: Can you please here mention it is the "clear" command other than that it looks good

**Review by Bishoy-at-pieces — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mdex-geek) [reply CHANGELOG.md]: hey i added  it 
can you look over

**Pull request merged**